project/Dependencies.scala

```diff
@@ -21,41 +21,52 @@ object Dependencies {
     val kinesisClient    = "1.9.1"
     val kinesisConnector = "1.3.0"
     val validator        = "2.2.6"
-    val elasticsearch    = "6.2.4"
+    val elasticsearch    = "6.3.2"
     val nsqClient        = "1.1.0-rc1"
+    val jackson          = "2.9.6"
+
+    // TODO: consider removing paranamer version override when upgrading other libaries
+    val paranamer = "2.8" // Jackson 2.9.6 uses version 2.8 of paranamer internally
+                          // but there is json4s-core_2.12 3.2.11 that overrides it back to 2.6
+
```

Comment:
`paranamer` was source of the Jackson issue. Jackson is used internally in `elastic4s` and not changable and Jackson uses `paranamer` for detecting fields and methods of a pojo. There is `json4s` dependency that I guess comes form snowplow analytics sdk. that one has older `paranamer` and caused the issue.